index.html

```diff
@@ -11,15 +11,24 @@
 
```

Comment:
In order for your site to be mobile responsive, you need to include the viewport meta tag in the head:

`<meta name="viewport" content="device-width, initial-scale=1">`